vtu11/impl/vtu11_impl.hpp

```diff
@@ -146,6 +207,38 @@ void write( const std::string& filename,
   output.close( );
 }
 
+//ParallelWrite generates a pvtu file and accordingly the vtu pieces in a subfolder
+//Each piece consists of a set of points and cells
+template<typename MeshGenerator, typename Writer>
+void parallelWrite( const std::string& path,
+                    const std::string& baseName,
+                    MeshGenerator& mesh,
+                    const std::vector<DataSet>& pointData,
+                    const std::vector<DataSet>& cellData,
+                    const size_t fileId,
+                    const size_t numberOfFiles,
+                    Writer writer)
+{
+  fs::path directory = path;
```

Comment:
```suggestion
  fs::path directory(path);
```

Reply:
Why?

Reply:
I am having problems with newer versions, then the implicit conversion btw `std::string` and `fs::path` is problematic in Win
apparently win uses `std::wstring` at some point

Reply:
also why not? :)

Reply:
Ah okay. Would that count as implicit conversion? To me those two are equivalent, but idk. Either way doesn't matter, I was just wondering.

Reply:
not sure, maybe would also work without ...

Reply:
Well we can use your suggested change just to be sure.

But to not confuse the others: Initialization with = is fine, it (as far as I know) is the same as using the "constructor syntax". It's not going into copy assignment.